network-api/networkapi/buyersguide/factory.py

```diff
@@ -60,29 +60,29 @@ def product_category(self, create, extracted, **kwargs):
             else:
                 return
 
-    company = Faker('company')
     blurb = Faker('sentence')
-    url = Faker('url')
-    price = LazyAttribute(lambda _: randint(49, 1500))
     camera_app = Faker('boolean')
-    meets_minimum_security_standards = Faker('boolean')
     camera_device = Faker('boolean')
-    microphone_app = Faker('boolean')
-    microphone_device = Faker('boolean')
+    company = Faker('company')
+    delete_data = Faker('boolean')
+    email = Faker('email')
+    live_chat = Faker('url')
     location_app = Faker('boolean')
     location_device = Faker('boolean')
-    uses_encryption = LazyFunction(get_extended_yes_no_value)
-    privacy_policy_reading_level_url = Faker('url')
+    manage_vulnerabilities = LazyFunction(get_extended_yes_no_value)
+    meets_minimum_security_standards = Faker('boolean')
+    microphone_app = Faker('boolean')
+    microphone_device = Faker('boolean')
+    parental_controls = LazyFunction(get_extended_yes_no_value)
```

Comment:
_actual_ change: switched from `Faker('boolean')` to the extended yes/no field generator.